package.json

```diff
@@ -177,6 +177,7 @@
     "react-addons-test-utils": "15.4.0",
     "react-hot-loader": "1.3.0",
     "react-test-env": "0.2.0",
+    "redux-mock-store": "1.2.1",
```

Comment:
If we plan on adding a new dev dependency, don't forget to update the shrinkwrap file. You can do this by navigating to your local checkout and typing `make shrinkwrap` which handles the process for you. To see what the script does see this [doc](https://github.com/Automattic/wp-calypso/blob/master/docs/shrinkwrap.md)